.github/dependabot.yml

```diff
@@ -23,3 +23,12 @@ updates:
     - dependabot
     commit-message:
       prefix: BAU
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
+    target-branch: main
+    labels:
+    - dependabot
+    commit-message:
+      prefix: BAU
```

Comment:
I think it would be worth defining a specific prefix for this kind of PR
e.g. `BAU - dependabot`